cmd/utils.go

```diff
@@ -831,3 +834,41 @@ func execAndWaitWithWorkDirReturnErr(command string, args []string, logger appso
 	}
 	return err
 }
+
+func checksum256TestFile(newFileName string, oldFileName string) (bool, error) {
+	var checkValue bool
+	fmt.Println("Checksum 256")
+	fmt.Println("oldFile", oldFileName)
+	fmt.Println("newFile", newFileName)
+
+	oldFile, err := os.Open(oldFileName)
+	if err != nil {
+		return false, errors.Errorf("File open failed for %s controller binary: %v", oldFileName, err)
+
+	}
+	defer oldFile.Close()
+
+	oldSha256 := sha256.New()
+	if _, err := io.Copy(oldSha256, oldFile); err != nil {
```

Comment:
Could we extract the bit about get checksum for a file in a separate util function? I see we do this twice here. and might make the code cleaner.